### ISSUE TITLE
Wrong LidarSensorView::Reflection

### ISSUE THREAD

**Issue by PhilippRosenbergerTUDarmstadtFZD** [OpenSimulationInterface/open-simulation-interface#260]
It seems like that the Reflection for LidarSensorView is a copy-paste mistake from Radar.
Time of Flight is fine.
We are missing  the angles of the ray and we do not need antenna diagrams or doppler information here.
Please change the following:
```Protobuf
    //
    // \brief Definition of the lidar reflection.
    //
    message Reflection
    {
        // Relative signal level of the reflection.
        //
        // This takes the combined antenna diagram (losses in TX and RX)
        // as well as the signal losses due to scattering and absorption
        // into account, and will, when multiplied by TX power yield the
        // actual RX power.
        //
        // Unit: [dB]
        //
        optional double signal_strength = 1;

        // Time of flight.
        //
        // This is the time of flight of the reflection, which is directly
        // proportional to the distance traveled.
        //
        // Unit: [s]
        //
        optional double time_of_flight = 2;

        // Doppler shift.
        //
        // Shift in frequency based on the specified TX frequency.
        //
        // Unit: [Hz]
        //
        optional double doppler_shift = 3;
    }
}
```

**Comment by pmai** [comment]
The angles of the rays are implict in their location in the repeated field (the same as for Radar, and similar to e.g. common Lidar packet structures).

The effect of "antenna" diagrams is included both for uniformity with radar  and to allow modeling of non-uniform detection/emittance patterns (consider arrayed lasers/detectors, or flash technology with CCD detectors and optics). Same for doppler information (allows e.g. modelling of certain speed related effects, or for new hybrid sensor designs, e.g. "optical radar").

**Comment by PhilippRosenbergerTUDarmstadtFZD** [comment]
O.K., thats fine, but I would still change the description for 
``
optional double osi3::LidarSensorView::Reflection::signal_strength = 1
``,
as we do not have antenna diagrams for lidar.

**Comment by pmai** [comment]
Yes that's true, we did not include the antenna diagrams in the current version (since strictly speaking only needed for multi-path propagation), so I agree that we either need to fix this or change the description. For 3.0.1 I'd lean to fixing the description.

**Comment by PhilippRosenbergerTUDarmstadtFZD** [comment]
Yes, I would also go for the description.

**Comment by LudwigFriedmannBMW** [comment]
Hello @

**Comment by LudwigFriedmannBMW** [comment]
Hello @PhilippRosenbergerTUDarmstadtFZD, could you provide a proposal for the extension of the documentation?

**Comment by LudwigFriedmannBMW** [comment]
With the fixes in the documentation in #265, I consider this task as completed.

**Comment by PhilippRosenbergerTUDarmstadtFZD** [comment]
We have double-checked and agree to closing the issue.
